src/main/shell-session.ts

```diff
@@ -125,6 +125,8 @@ export class ShellSession extends EventEmitter {
     if (this.preferences.httpsProxy) {
       env["HTTPS_PROXY"] = this.preferences.httpsProxy
     }
+    const no_proxy = ["localhost", "127.0.0.1", env["NO_PROXY"]]
+    env["NO_PROXY"] = no_proxy.filter(address => !!address).join()
```

Comment:
this leaves a trailing comma if `NO_PROXY` was not already defined:
```
jimehrismann@JE-MBP13-2017:~$ env | grep NO_PROXY
NO_PROXY=localhost,127.0.0.1,
jimehrismann@JE-MBP13-2017:~$
```
presumably that's not an issue?

Reply:
Why is it leaving that? 🤔 I thought that filter would have rejected empty values

Reply:
maybe `env["NO_PROXY"]` is filled with whitespace

Reply:
this might impact the changes in kubectl.ts as well?

Reply:
never mind, sorry. I checked out `fix/skip-https-proxy-for-localhost` but forgot to pull after. So I got Jari's commit but not yours Lauri. After pulling I do not see the trailing comma.
@nevalla 

But it could still be there in the kubectl.ts changes(?)